### PR TITLE
Cherry-pick commits for 0.13.3 release

### PR DESCRIPTION
**Pull request by songy23**
Fix several issues that negatively affect user experience (most of which are dependency issues).

Please let me know if there are other commits that you want to include in this patch release.

### CODE REVIEW

**Comment by songy23** [comment]
FYI don't worry about the failed build targets - Kokoro is not enabled on this branch.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,12 @@
 ## Unreleased
+- Adds Tracing.getExportComponent().shutdown() for use within application shutdown hooks.
+- `Duration.create` now throws an `IllegalArgumentException` instead of
+  returning a zero `Duration` when the arguments are invalid.
+- `Timestamp.create` now throws an `IllegalArgumentException` instead of
+  returning a zero `Timestamp` when the arguments are invalid.
```
Comment (sebright) [inline CHANGELOG.md]: Did you add these three entries while resolving a merge conflict?  They are not in the original commit.

Comment (songy23) [reply CHANGELOG.md]: Removed https://github.com/census-instrumentation/opencensus-java/commit/b647dbda2e7131d7d350f63d28943c93fe94fc47, that one is less an issue and may be better put into next minor release.

**Comment by songy23** [comment]
Chatted offline, we think it's better to do a 0.14.0 minor release instead. Will do on Monday.

**Pull request closed without merging**